static/index.html

```diff
@@ -101,16 +119,16 @@
 
             <div class="hint">
                 <p class="summary">Example event query with sub-string filter over payload property</p>
-                <a href="http://eventcube.red-gate.com/1.0/event?expression=download_started(payload).re(payload.productName,'SQL')&start=2015-07-19T22:00:00Z&stop=2015-07-20T00:00:00Z">
-                    download_started(payload).re(payload.productName,'SQL')&start=2015-07-19T22:00:00Z&stop=2015-07-20T00:00:00Z
+                <a href="http://eventcube.red-gate.com/1.0/event?expression=download_started(payload).re(payload.productName,'SQL')&amp;start=2015-07-19T22:00:00Z&amp;stop=2015-07-20T00:00:00Z">
+                    download_started(payload).re(payload.productName,'SQL')&amp;start=2015-07-19T22:00:00Z&amp;stop=2015-07-20T00:00:00Z
                 </a>
                 <p class="description">The .re filter is a actually a regular expression filter, but without any regex syntax it acts as a sub-string filter.</p>
             </div>
 
             <div class="hint">
                 <p class="summary">Example metric query</p>
-                <a href="http://eventcube.red-gate.com/1.0/metric?expression=distinct(embargoing_failure(payload.ipAddress).re(payload.message,'determine country'))&step=864e5&start=2015-07-15T00:00:00Z&stop=2015-07-20T00:00:00Z">
-                    distinct(embargoing_failure(payload.ipAddress).re(payload.message,'determine country'))&step=864e5&start=2015-07-15T00:00:00Z&stop=2015-07-20T00:00:00Z
+                <a href="http://eventcube.red-gate.com/1.0/metric?expression=distinct(embargoing_failure(payload.ipAddress).re(payload.message,'determine%20country'))&amp;step=864e5&amp;start=2015-07-15T00:00:00Z&amp;stop=2015-07-20T00:00:00Z">
```

Comment:
Not sure about this one (escaping space as %20 in the URL), some people suggest space is escaped as '+' in the query string.